### PR TITLE
Am/improved send phonons

### PR DESCRIPTION
**Pull request by italianst4**
This resolves the BSOD (black screen of death) when trying to send a Phonon.  Additionally, I added visual loading states for pairing and sending a Phonon.  I also added a message which provides context to a known issue with pairing the same two remote cards more than once.

### CODE REVIEW

File: src/components/SendPhononButton.tsx [hunk 1]
```diff
@@ -1,40 +1,26 @@
...
 import SendPhononModal from "./SendPhononModal";
 
 const SendPhononButton: React.FC<{
-  phonon: PhononDTO;
   selectedPhonon: PhononDTO | undefined;
   setSelectedPhonon: React.Dispatch<
     React.SetStateAction<PhononDTO | undefined>
   >;
-}> = ({ phonon, selectedPhonon, setSelectedPhonon }) => {
+  isConnectedToServer: boolean;
```
Comment (douglance) [inline src/components/SendPhononButton.tsx:12]: :eyes:

File: src/components/SendPhononModal.tsx [hunk 2]
```diff
@@ -14,52 +16,79 @@ export type SendPhononFormData = {
...
+          if (
+            String(err.data).trim().toLowerCase() ===
+            String("timeout").toLowerCase().trim()
+          ) {
+            setErrorMessageDetails(
+              "There's a known issue that is preventing the pairing of a remote card after a previous Phonon was sent to that same remote card. This will be resolved in a future release."
+            );
+          }
+
+          // console.error(err);
```
Comment (douglance) [inline src/components/SendPhononModal.tsx:87]: cruft

File: src/components/SendPhononModal.tsx [hunk 3]
```diff
@@ -68,46 +97,92 @@ const SendPhononModal: React.FC<{
...
-              size="large"
-              expand="full"
-              fill="clear"
-              color="medium"
-              onClick={destroyModal}
-              disabled={isLoading}
-            >
-              CANCEL
-            </IonButton>
+        {isPairing ? (
```
Comment (douglance) [inline src/components/SendPhononModal.tsx:112]: this logic is tough to follow: 

```ts
isPairing 
	? <pairing spinner> 
	: isSending 
		? <sending spinner> 
		: isConnectedToServer 
			? <send form>
			: <close> 
```
The logic might be easier to follow like this: 

```ts
isPairing && <paring spinner>
isSending && <sending spinner>
!isPairing && !isSending && isConnectedToServer 
	? <send form>
	: <close>
```

assuming isPairing/isSending are never true at the same time. 

Also, to keep it concise in the returned JSX, I might consider pulling each of those <chunks> into their own mini component inside this one, e.g. :

```ts
const PairingSpinner = () => ( 
	<div className="text-xl mt-8 mx-auto flex items-center">
		<IonSpinner className="mr-2" />
		Pairing with{" "}
		<span className="font-bold ml-1">{getValues("cardId")}</span>...
	</div>
)

const SendingSpinner = () => (
 	<div className="text-xl mt-8 mx-auto flex items-center">
      <IonSpinner className="mr-2" />
      Sending{" "}
      <span className="font-bold ml-1">
        {isGreaterThan(selectedPhonon.Denomination, 0) ? (
          isNativePhonon(selectedPhonon) ? (
            selectedPhonon.Denomination
          ) : (
            weiToEth(selectedPhonon.Denomination)
          )
        ) : (
          <IonSpinner />
        )}{" "}
        {chain?.ticker}
      </span>{" "}
      to
      <span className="font-bold ml-1">{getValues("cardId")}</span>...
    </div>
)

// etc.
```

**Review by douglance — APPROVED** [review]

File: src/components/SendPhononModal.tsx (hunk 2 above)
Comment (italianst4) [reply src/components/SendPhononModal.tsx:87]: goodbye

File: src/components/SendPhononButton.tsx (hunk 1 above)
Comment (italianst4) [reply src/components/SendPhononButton.tsx:12]: just observing?

**Pull request merged**